.github/workflows/main.yml

```diff
@@ -29,7 +29,7 @@ jobs:
       matrix:
         # Latest stable version, update at will
         os: [ macOS-10.15, ubuntu-18.04, windows-2019 ]
-        dc: [ dmd-2.093.1, ldc-1.23.0, dmd-master, ldc-master ]
+        dc: [ dmd-latest, ldc-latest, dmd-master, ldc-master ]
```

Comment:
Isn't it better to use a fixed release? #2105 is failing and I don't know if the problem is with my PR or because the test is now using a different compiler.

Reply:
I suppose, but it fell several releases behind when it wasn't pegged to the latest release. Also dmd is tested upstream so *hopefully* the it should be on the PR not the compiler (in theory...)